internal/gcloud/cloud_run_service.go

```diff
@@ -41,7 +41,7 @@ func (s CloudRunService) Delete(sampleDir string) error {
 	_, err := util.ExecCommand(exec.Command("gcloud", a...), sampleDir)
 
 	if err != nil {
-		return fmt.Errorf("[CloudRunService.delete] deleting Cloud Run Service: %w", err)
+		return fmt.Errorf("util.ExecCommand deleting Cloud Run Service: %w", err)
```

Comment:
I would say that these and the other util.ExecCommand errors don't need to include the method name. The error messages that you have are what's most useful here and ExecCommand is just a thin wrapper. There's some room for debate here probably but that's my inclination.

Reply:
Agreed given there's not a lot of different ways to exec in this project.

I often find it is useful in programs which passthru and execute shell commands to actually log the command used, especially in an error. However, that is a further logging enhancement out of scope of this PR.

Reply:
Resolved, thank you.